src/stormpath.auth.js

```diff
@@ -135,6 +135,7 @@ angular.module('stormpath.auth',['stormpath.CONFIG'])
         op.then(function(){
           $rootScope.$broadcast(STORMPATH_CONFIG.SESSION_END_EVENT);
         },function(response){
+          $rootScope.$broadcast(STORMPATH_CONFIG.SESSION_END_EVENT);
```

Comment:
One thing that might cause this is for example when the user's session has timed out. When they try to logout, the server returns an error that the token is invalid and the user can not logout from the UI.